app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -144,18 +171,37 @@ class PatcherViewModel(
                                 input.selectedPatches
                             )
                         }
-                    } else {
-                        app.toast(app.getString(R.string.install_app_fail, extra))
+                    }
+
+                    installerStatusDialogModel.packageInstallerStatus = pmStatus
```

Comment:
@PalmDevs should the dialog be shown for successful installs?

Reply:
If not, after installation the only thing that changes is the button from "Install" to "Ok"

Reply:
If the package installer dialog is hidden after the user clicks Install (some apps do that), yes. Otherwise, maybe not considering the package manager dialog already shows the same thing.

Reply:
The popup is hidden once the user clicks install.

Reply:
Should this be marked as resolved?